app/routes/upload-photos.tsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from "@remix-run/react";
+
+export default function Requirements() {
```

Comment:
This probably doesn't matter but I'll correct the name (it might improve error messages? Or we could just leave it out and keep the function unnamed)

```suggestion
export default function UploadPhotos() {
```